dazel/lib/src/bazelify/build.dart

```diff
@@ -165,7 +163,7 @@ class BuildFile {
     final buildersUsed =
         libraries.expand((l) => l.builders?.keys ?? const <String>[]);
     for (var builder in buildersUsed) {
-      var builderDefinition = buildConfigs.values
+      var builderDefinition = buildConfigs.dependencies.values
```

Comment:
Tracked this down. When we create the `BuildConfig` instances we make a distinction between the local package and dependencies. We create 2 versions of the BuildConfig for the local package, 1 as if it was a dependency and 1 as if it's local, and we wend up relying on the order they are inserted into the Map for the 'local' version of it to end up winning.

We could go back to something more along these lines, since in usage now I only end up doing that logic manually rather than have it be implicit in the order of work... however I think I prefer to keep it more explicit and we might end up later having other reason to make a more obvious distinction between local vs dependencies.